apps/src/code-studio/pd/workshops/types.ts

```diff
@@ -46,22 +48,110 @@ export interface GetWorkshopInfoScriptDataResponse {
   facilitators?: FacilitatorInfo[];
 }
 
-export type UserInfoForWorkshop = {
+export interface WorkshopInfo {
+  id: number;
+  course: string;
+  subject?: string;
+  courseOfferings?: string[];
+  name?: string;
+  capacity: number;
+  numEnrollments: number;
+  gradeLevels?: string[];
+  sessions: SessionInfo[];
+  format: keyof typeof WorkshopFormats;
+  locationName?: string;
+  fee?: string;
+  prereq?: string;
+  description?: string;
+  notes?: string;
+  customRegistrationLink?: string;
+  regionalPartnerName?: string;
+  organizer: OrganizerInfo;
+  facilitators?: FacilitatorInfo[];
+}
+
+export const workshopInfoDataResponseToParams = (
+  response: GetWorkshopInfoScriptDataResponse | null
+) => {
```

Comment:
```suggestion
): WorkshopInfo | null => {
```

defining the return type here isn't required since it can usually be inferred, but I'm guessing the casting on line 98 is forcing something... 🤔 

if you remove the casting and add this return type, does the compiler say something isn't right?